src/Components/Shared/Modal/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styles from './modal.module.css';
+
+const SharedModal = (props) => {
```

Comment:
We could pass the props in this way: Shared Modals = ({ data, handleClick, etc }), i think it'll be easier to read